modules/ROOT/pages/clustering/servers.adoc

```diff
@@ -84,8 +84,8 @@ Once enabled, the server may be allocated databases to host.
 
 When a server is no longer needed, it cannot be removed from the cluster while it is still allocated to host any databases.
```

Comment:
I think we should shift the focus of this section to the state rather than the command for deallocating databases. How about something like this:
```suggestion
A server in a _deallocating_ state is transitioning to a state where it can no longer host databases.
This means that a server is no longer needed, and you want to remove it from the cluster.
```